examples/example-detector.ipynb

```diff
@@ -511,7 +511,7 @@
    "outputs": [],
    "source": [
     "model = Model().to(device)\n",
-    "optim = torch.optim.SGD(model.parameters(), lr=1e-02, weight_decay=1e-04, momentum=0.9)"
+    "optim = torch.optim.Adam(model.parameters(), lr=1e-04, weight_decay=5e-04)"
```

Comment:
```suggestion
    "optim = torch.optim.Adam(model.parameters(), weight_decay=5e-04)"
```

I think default LR is usually the way to go with Adam

Reply:
:+1: shouldn't really matter in `1e-03` vs `1e-04` but will take the change